app/helpers/banners_helper.rb

```diff
@@ -3,5 +3,29 @@ module BannersHelper
   def has_banners?
     @banners.present? && @banners.count > 0
   end
-
+  
+  def is_banner_on_top?
+    @banners.sample.banner_position == "Top"
+  end
+  
+  def is_banner_on_bottom?
+    @banners.sample.banner_position == "Bottom"
+  end
+  
+  def is_banner_heading_on_top?(heading_id)
+    if (@banners.sample.headings != nil) && (@banners.sample.headings.include? heading_id.to_s)
```

Comment:
Why are we using `sample`? Are we assuming all banners have the same headings and the same position?